gulp/constants.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-env es6 */
+'use strict';
+
+// External dependencies
+import requireUncached from 'require-uncached';
+export const gulpPlugins = require('gulp-load-plugins')();
+
+// gulp string replace options
+export const gulpReplaceOptions = {
+    logs: {
+      enabled: false
+    },
+    searchValue: 'string',
```

Comment:
The `searchValue` option is problematic.  With it set, we are not replacing the text-domains or function prefixes in PHP.  Whoops.  By removing it, everything replaces as it should.

Reply:
d'oh! I updated it to explicitly be regex rather than deleting the value in case the default changes in the future.